Use 1e10 instead of 10000000000

AAYUSH-GUPTA-coder opened the pull request:
In PriceConverter.sol smart contract. Use 1e10 instead of 10000000000 and  1e18 instead of 1000000000000000000 for better understanding and readability of code.

AAYUSH-GUPTA-coder committed:
Use 1e10 instead of 10000000000

In PriceConverter.sol smart contract. Use 1e10 instead of 10000000000 and  1e18 instead of 1000000000000000000 for better understanding and readability

AAYUSH-GUPTA-coder committed:
Use 1e10 instead of 10000000000

In PriceConverter.sol smart contract. Use 1e10 instead of 10000000000 and 1e18 instead of 1000000000000000000 for better understanding and readability of code.

anannyenaik reviewed (changes requested):
Hey, I agree that it will be much easier for beginners to follow, but I am speculating that the current approach is primarily for practice.

Maybe comment out the changes and Patrick can decide the one to keep when he reviews the PR.

PatrickAlphaC commented:
^ What @krakxn said.
Thanks!

AAYUSH-GUPTA-coder committed:
comment out the changes. Patrick can decide.

comment out the changes and Patrick can decide the one to keep when he reviews the PR. As discussed in GitHub conversation.

AAYUSH-GUPTA-coder commented:
Done ! comment out the changes and Patrick can decide the one to keep when he reviews the PR.
